Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -470,5 +479,17 @@ namespace AZ
             return m_multisampleState;
         }
 
+        void RPISystem::RegisterXRSystem(XRRenderingInterface* xrSystemInterface)
+        { 
+            AZ_Assert(!m_xrSystem, "XR System is already registered");
+            m_xrSystem = xrSystemInterface;
+            m_rhiSystem.RegisterXRSystem(dynamic_cast<RHI::XRRenderingInterface*>(xrSystemInterface));
+        }
+
+        void RPISystem::UnRegisterXRSystem()
```

Comment:
Nit: prefer `Unregister` (as this is a word, no need for the capital `R`)

Reply:
Will address this as part of next work